javalin-plugins/javalin-swagger-plugin/src/test/kotlin/io/javalin/openapi/plugin/swagger/SwaggerPluginTest.kt

```diff
@@ -48,4 +48,117 @@ internal class SwaggerPluginTest {
         }
     }
 
-}
\ No newline at end of file
+    @Test
+    fun `should not fail if second swagger plugin is registered`(){
+        val swaggerConfiguration = SwaggerConfiguration();
+        val otherConfiguration = ExampleSwaggerPlugin();
+        Javalin.create{
+            it.plugins.register(SwaggerPlugin(swaggerConfiguration))
+            it.plugins.register(otherConfiguration)
+        }
+            .start(8080)
+            .use{
+                val javalinHost = "http://localhost:8080"
+
+                val webjarCssRoute = "/webjars/swagger-ui/${swaggerConfiguration.version}/swagger-ui.css"
+                val webjarJsRoute = "/webjars/swagger-ui/${swaggerConfiguration.version}/swagger-ui-bundle.js"
+                val webjarJsStandaloneRoute = "/webjars/swagger-ui/${swaggerConfiguration.version}/swagger-ui-standalone-preset.js"
```

Comment:
I think it'd be enough to test just one resource here, in the end all of them come from the same webjar bundle